modules/xmpp/JingleSessionPC.js

```diff
@@ -1835,6 +1842,12 @@ export default class JingleSessionPC extends JingleSession {
         if (!FeatureFlags.isSsrcRewritingSupported()) {
             return;
         }
+
+        if (mediaType === MediaType.AUDIO && this.options.startSilent) {
+
+            return;
```

Comment:
is the session going to eventually catch up with the skipped sources? or does startSilent mean no audio for the entire session?

Reply:
The latter. A full session restart is necessary for getting the missing sources.